BookingsApi/BookingsApi.Contract/V2/Responses/HearingDetailsResponseV2.cs

```diff
@@ -111,4 +112,10 @@ public class HearingDetailsResponseV2
         /// The group id for a hearing
         /// </summary>
         public Guid? GroupId { get; set; }
+        
+        /// <summary>
+        /// List of judiciary participants in a hearing
+        /// </summary>
+        [JsonProperty("judicial_office_holders")]
```

Comment:
personally I like JudiciaryParticipants (that's what we call it internally) but Murali requested judicial_office_holders as that is what is in the API schema. I've tried to avoid calling it JudicialOfficeHolders in code as that can be confused with the existing JudicialOfficeHolder entity (which doesn't include judges)

Reply:
no problem